source/kubernetes/kubectl-plugin-reference.rst

```diff
@@ -0,0 +1,620 @@
+=================================
+MinIO Kubernetes Plugin Reference
```

Comment:
@nitisht this is a first pass on the Kubernetes Plugin docs. I still need to build out the Quick Reference and Common Operations, but I would appreciate you taking a look - especially at the Syntax section.